app/models/form/sales/questions/shared_ownership_type.rb

```diff
@@ -4,13 +4,20 @@ def initialize(id, hsh, page)
     @id = "type"
     @check_answer_label = "Type of shared ownership sale"
     @header = "What is the type of shared ownership sale?"
-    @hint_text = "A shared ownership sale is when the purchaser buys up to 75% of the property value and pays rent to the Private Registered Provider (PRP) on the remaining portion"
     @top_guidance_partial = guidance_partial
     @type = "radio"
     @answer_options = answer_options
     @question_number = 4
   end
 
+  def hint_text
+    if form.start_year_after_2024?
+      "When the purchaser buys an initial share of up to 75% of the property value and pays rent to the Private Registered Provider (PRP) on the remaining portion, or a subsequent staircasing transaction"
```

Comment:
nit: I think we've been adding full stops to the end of hint texts (though I see this wasn't there in the previous version either)